src/app/modules/sgid/sgid.service.ts

```diff
@@ -167,14 +177,16 @@ export class SgidServiceClass {
   }: {
     accessToken: string
   }): ResultAsync<
-    { sub: string; data: { 'myinfo.nric_number': string } },
+    { sub: string; data: SGIDScopeToValue },
```

Comment:
To update the doc for this method also since retrieveUserInfo is no longer just for NRIC